lib/compiler/compileSolcjs.ts

```diff
@@ -15,21 +16,14 @@ async function loadCompiler(config: Config) {
 }
 
 export function compileSolcjs(config: Config) {
-  return async function compile(sources: string[], findImports: (file: string) => any) {
+  return async function compile(sources: ImportFile[], findImports: (file: string) => any) {
     const solc = await loadCompiler(config);
-    const inputs = findInputs(sources);
-    const input = buildInputObject(convertInputs(inputs));
+    const input = buildInputObject(buildSourcesObject(sources));
```

Comment:
This could be simplified to just `buildInputObject(sources)` by having `buildInputObject` call `buildSourcesObject`